.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "semi": false,
+    "trailingComma": "none",
+    "singleQuote": false,
+    "printWidth": 80,
+    "quoteProps": "consistent"
+}
```

Comment:
Missing new line.

Reply:
So good news about prettier, it applies that rule out-the-box. The follow-up PR will correct this across the entire codebase.